arangod/Aql/ExecutionNode/EnumerateNearVectorNode.cpp

```diff
@@ -108,13 +117,13 @@ ExecutionNode* EnumerateNearVectorNode::clone(ExecutionPlan* plan,
                                               bool withDependencies) const {
   auto c = std::make_unique<EnumerateNearVectorNode>(
       plan, _id, _inVariable, _oldDocumentVariable, _documentOutVariable,
-      _distanceOutVariable, _limit, collection(), _index);
+      _distanceOutVariable, _limit, _ascending, _offset, collection(), _index);
   CollectionAccessingNode::cloneInto(*c);
   return cloneHelper(std::move(c), withDependencies);
 }
 
 CostEstimate EnumerateNearVectorNode::estimateCost() const {
-  // TODO
+  // TODO(jbajic) check if it is accurate
   CostEstimate estimate = _dependencies.at(0)->getCost();
   estimate.estimatedNrItems *= _limit;
   return estimate;
```

Comment:
I will do as you suggest, but I think this estimate probably needs to include the nLists parameter as well. Will leave a todo to update this in a better way since the nLists might be changed with something else